app/src/main/java/com/example/tictactoe/GameFragment.kt

```diff
@@ -13,10 +17,25 @@ import android.view.ViewGroup
  */
 class GameFragment : Fragment(R.layout.fragment_game) {
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+    private lateinit var gameGridView: GridView
+    private lateinit var gameGridAdapter: GridAdapter
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        gameGridView = view.findViewById(R.id.game_grid)
+        gameGridAdapter = GridAdapter(requireContext(), getInitialGameData())
+        gameGridView.adapter = gameGridAdapter
+        gameGridView.onItemClickListener =
+            AdapterView.OnItemClickListener { adapterView, view, position, id ->
+                // TODO(): Handle click listeners
+                val msg = "Position=$position, Id=$id"
+                Log.d("Game", msg)
```

Comment:
Remove log for now, will handle all initial logging as part of different issue. 

Created issue for this : [issues/14](https://github.com/SurajBichkunde39/TicTacToe/issues/14)

Reply:
Done